reconcile/utils/ocm.py

```diff
@@ -633,12 +633,35 @@ def get_kafka_clusters(self, fields=None):
                         for cluster in clusters]
         return clusters
 
+    def get_kafka_service_accounts(self, fields=None):
+        """Returns details of the Kafka service accounts """
+        results = []
+        api = '/api/managed-services-api/v1/serviceaccounts'
+        service_accounts = self._get_json(api)['items']
+        for sa in service_accounts:
+            sa_id = sa['id']
+            id_api = f'{api}/{sa_id}'
+            sa_details = self._get_json(id_api)
+            item = {k: v for k, v in sa_details.items() if k in fields}
```

Comment:
What happens if `fields` has its default value of `None`? Won't this crash?

Reply:
good catch! fixed